blog/content/post/2020-06-15-qubes_summation.md

```diff
@@ -0,0 +1,184 @@
+  - QubesOS
+  - minisummit
+  - live
+  - YouTube
+  - firmware
+  - coreboot
+  - security
+  - TPM
+categories:
+  - Miscellaneous
+  - Security
+  - Firmware
+
+
+---
+The second Qubes OS and 3mdeb 'minisummit' is ahead of us. We have gone through
+four evenings of topics devoted to Qubes OS. It has been another fruitful and
+productive event, during which we have exchanged loads of knowledge and
+expertise. Below you will find list of topics, we have discussed with related
+videos and broad report of each day.
+
+
+* 20.05
+  1. Michał Żygowski: Qubes OS on modern AMD platform
+    * [recording](https://www.youtube.com/watch?v=Rw7rAPPyPPc&t=31s)
+  2. Norbert Kamiński: Status fwupd/LVFS support for Qubes OS
+    * [recording](https://www.youtube.com/watch?v=o_IdERo3aiE&t=984s)
+  3. AMA session
+    * [recording](https://www.youtube.com/watch?v=BSGUcW6QDYU&t=1509s)
+
+* 28.05
+  1. Piotr Król: SRTM for Qubes OS VMs
+    * [recording](https://www.youtube.com/watch?v=Eip5Rts6S2I&t=2s)
+  2. Michał Żygowski: Anti Evil Maid for modern AMD UEFI-based platform
+    * [recording](https://youtu.be/rM0vRi6qABE?t=3)
+  3. AMA session
+    * [recording](https://youtu.be/rM0vRi6qABE?t=1904)
+
+* 04.06
+  1. Piotr Król: DRTM for Qubes OS VMs
+    * [recording](https://youtu.be/pZF-jyJWTE4)
+  2. Michał Żygowski: Anti Evil Maid for Intel coreboot-based platform
+    * [recording](https://youtu.be/YE2FbFlszI4?t=9)
+  3. AMA session
+    * [recording](https://youtu.be/YE2FbFlszI4?t=1725)
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+overview
+  2. Marek Marczykowski-Górecki: Operating system testing, when it itself uses
+  virtual machines
+  3. AMA session
+
+### Qubes OS future: AMD SEV and fwupd/LVFS
+
+ The first day of 'minisummit' gathered us around Qubes OS future: AMD SEV and
+fwupd/LVFS, where prelegents were discussing projects based on Qubes OS. The
+opening presetation was held by Michał Żygowski, Firmware Engineer in 3mdeb,
+Braswell SoC, PC Engines and Protectli maintainer in coreboot who has presented
+Qubes on SuperMicro M11SDV-4C-LN4F superserver platform, that has been tested
+with Qubes OS on board. Michał has introduced and explained it’s features,
+Secure Encrypted Visualisation (SEV) and Secure Memory Encryption (SME) with
+encrypted state (SEV-ES) extension and explained why SME is better than TSME.
+Afterwards, we have been introduced with status of mentioned features on Open
+Source, debating whether it is suppoorted, whether it can be used, what impact
+Qubes OS will have on the features or is it possible to run SEV-enabled guests
+using open-source tools. Michał described briefly also OVMF and SUSE Linux
+Enterprise Server support for SEV, Automatic Exits (AE) and Non-automatic Exits
+(NAE). After prelection, our specialists took the floor debating over host
+system protection, memory encryption on the BIOS level, XEN support and many
+more issues that can be found in the video above.
```

Comment:
@KBanecka I think we should consider 2 things for each presentation

- how community can engage
- what we hope to do in next 12 months

I believe you can ask authors of presentations